docs/user-guide/connecting-applications.md

```diff
@@ -425,6 +425,18 @@ $ curl https://162.22.184.144 -k
 The IP address in the `EXTERNAL_IP` column is the one that is available on the public internet.  The `CLUSTER_IP` is only available inside your
 cluster/private cloud network.
 
+Note that on AWS, type `LoadBalancer` creates an ELB, which uses a (long)
+hostname, not an IP.  It's too long to fit in the standard `kubectl get svc`
+output, in fact, so you'll need to do `kubectl describe service nginxsvc` to
+see it.  You'll see something like this:
```

Comment:
Can we just modify the kubectl client to print something like <check describe output> for status? Can we detect the lb is on aws?